.github/workflows/ci.yml

```diff
@@ -113,7 +113,7 @@ jobs:
     - name: Cache Dependencies
       uses: coursier/cache-action@v6
     - name: Generate Readme
-      run: sbt docs/generateReadme
+      run: sbt zio-bson-docs/generateReadme
```

Comment:
ci.yml is autogenerate, so do not edit it manually.
the workaround is to rename the `zio-bson-doc` to `docs` in build.sbt and run `sbt ciGenerateGithubWorkflow`
I'll try to fix this issue on zio-sbt plugin.

Reply:
done